.github/workflows/test.yml

```diff
@@ -2,11 +2,16 @@ name: CI
 
 on:
   push:
-  pull_request:
-  workflow_dispatch:
+    branches:
+      - eth-max-vault
+      - bnb-max-vault
+  pull_request:  # Runs on all branches
+  workflow_dispatch:  # Manual trigger
 
 env:
   FOUNDRY_PROFILE: ci
+  ETH_MAINNET_RPC_URL: ${{ secrets.ETH_MAINNET_RPC_URL }}
```

Comment:
why is this here? this is the bnb branch

Reply:
removed it